app/models.py

```diff
@@ -1670,7 +1670,7 @@ def formatted_status(self):
                 "delivered": "Delivered",
                 "sending": "Sending",
                 "created": "Sending",
-                "sent": "Sent internationally",
```

Comment:
I believe it was set to this value because in the past, without delivery receipts, the status was `sending` and not `sent`. Anyway this does not make sense anymore since we can have `sent` to `delivered`